_layouts/default.html

```diff
@@ -29,5 +29,8 @@
     <div id="content">{{ content }}</div>
     {% include footer.html %}
     {% include google-analytics.html %}
+    <!-- Start of HubSpot Embed Code -->
+    <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/5557732.js"></script>
+    <!-- End of HubSpot Embed Code -->
```

Comment:
I think we should implement this like we did with GA and create an includes file, that way if it needs to update it is just one place to change it.